docs/rancher.md

```diff
@@ -0,0 +1,320 @@
+gcloud compute ssh --zone ${ZONE} worker2 -- sudo vgcreate myvg /dev/nvme0n1
+```
+
+Install `lvmd` on `worker1` and `worker2`.
+
+```bash
+gcloud compute ssh --zone ${ZONE} worker1
+
+# Install lvmd
+TOPOLVM_VERSION=0.4.0
+sudo mkdir -p /opt/sbin
+curl -sSLf https://github.com/cybozu-go/topolvm/releases/download/v${TOPOLVM_VERSION}/lvmd-${TOPOLVM_VERSION}.tar.gz | sudo tar xzf - -C /opt/sbin
+
+# Register service
+sudo curl -sSL -o /etc/systemd/system/lvmd.service https://raw.githubusercontent.com/cybozu-go/topolvm/v${TOPOLVM_VERSION}/deploy/systemd/lvmd.service
+sudo systemctl enable lvmd
+sudo systemctl start lvmd
+exit
+```
+
+```bash
+gcloud compute ssh --zone ${ZONE} worker2
+
+# Install lvmd
+TOPOLVM_VERSION=0.4.0
+sudo mkdir -p /opt/sbin
+curl -sSLf https://github.com/cybozu-go/topolvm/releases/download/v${TOPOLVM_VERSION}/lvmd-${TOPOLVM_VERSION}.tar.gz | sudo tar xzf - -C /opt/sbin
+
+# Register service
+sudo curl -sSL -o /etc/systemd/system/lvmd.service https://raw.githubusercontent.com/cybozu-go/topolvm/v${TOPOLVM_VERSION}/deploy/systemd/lvmd.service
+sudo systemctl enable lvmd
+sudo systemctl start lvmd
+exit
+```
+
+## 5. Deploy TopoLVM
+
+
+```bash
+TOPOLVM_VERSION=0.4.0
+kubectl apply -k https://github.com/cybozu-go/topolvm/deploy/manifests?ref=v${TOPOLVM_VERSION}
+kubectl apply -f https://raw.githubusercontent.com/cybozu-go/topolvm/v${TOPOLVM_VERSION}/deploy/manifests/certificates.yaml
+```
+
+## 6. Configure `topolvm-scheduler`
+
+You almost finish deploying TopoLVM here, but you need a little tweak to make `topolvm-scheduler` work properly.
+
+###  Update `topolvm-scheduler` manifest
+
+First, `master` has the following label and taint.
+
+- Label
+    1. `node-role.kubernetes.io/controlplane=true`
+    2. `node-role.kubernetes.io/etcd=true`
+- Taints
+    1. `node-role.kubernetes.io/controlplane=true:NoSchedule`
+    2. `node-role.kubernetes.io/etcd=true:NoExecute`
+
+To locate `topolvm-scheduler` onto `master`, update node affinity and toleration.
+
+```bash
+$ kubectl edit daemonset topolvm-scheduler -n topolvm-system
+# Edit as follows
+ apiVersion: apps/v1
+ kind: DaemonSet
+...
+ spec:
+...
+   template:
+...
+     spec:
+       affinity:
+         nodeAffinity:
+           requiredDuringSchedulingIgnoredDuringExecution:
+             nodeSelectorTerms:
+             - matchExpressions:
+-              - key: node-role.kubernetes.io/master
++              - key: node-role.kubernetes.io/controlplane
+                 operator: Exists
+...
+       tolerations:
+       - key: CriticalAddonsOnly
+         operator: Exists
+-      - effect: NoSchedule
+-        key: node-role.kubernetes.io/master
++      - key: node-role.kubernetes.io/controlplane
++        operator: Exists
++      - key: node-role.kubernetes.io/etcd
++        operator: Exists
+...
+```
+
+### Apply scheduler extension
+
+Download the scheduler extension configuration files onto `master`.
```

Comment:
Please write that we need to put the config file under `/etc/kubernetes` and that reason.